io.freetubeapp.FreeTube.metainfo.xml

```diff
@@ -3,15 +3,411 @@
+  <summary xml:lang="be">Праграма YouTube з адкрытым зыходным кодам для забеспячэння прыватнасці</summary>
+  <summary xml:lang="bg">Приложение за YouTube с отворен код за поверителност</summary>
+  <summary xml:lang="bn">গোপনীয়তার জন্য একটি ওপেন-সোর্স YouTube অ্যাপ</summary>
+  <summary xml:lang="ca">Una aplicació de YouTube de codi obert per a la privadesa</summary>
+  <summary xml:lang="cs">Open-source aplikace YouTube pro ochranu soukromí</summary>
+  <summary xml:lang="da">En open source YouTube-app til beskyttelse af personlige oplysninger</summary>
+  <summary xml:lang="de">Eine Open-Source-YouTube-App für den Datenschutz</summary>
+  <summary xml:lang="el">Μια εφαρμογή YouTube ανοιχτού κώδικα για προστασία απορρήτου</summary>
+  <summary xml:lang="en_AU">An open-source YouTube app for privacy</summary>
+  <summary xml:lang="en_GB">An open-source YouTube app for privacy</summary>
+  <summary xml:lang="en">An open-source YouTube app for privacy</summary>
+  <summary xml:lang="eo">Malfermfonteca YouTube-aplikaĵo por privateco</summary>
+  <summary xml:lang="es">Una aplicación de YouTube de código abierto para la privacidad</summary>
+  <summary xml:lang="et">Avatud lähtekoodiga YouTube'i rakendus privaatsuse tagamiseks</summary>
+  <summary xml:lang="fa">یک برنامه یوتیوب متن‌باز برای حفظ حریم خصوصی</summary>
+  <summary xml:lang="fi">Avoimen lähdekoodin YouTube-sovellus yksityisyyden suojaamiseksi</summary>
+  <summary xml:lang="fil">Isang open-source na YouTube app para sa privacy</summary>
+  <summary xml:lang="fr">Une application YouTube open source pour la confidentialité</summary>
+  <summary xml:lang="ga">Aip YouTube foinse oscailte le haghaidh príobháideachta</summary>
+  <summary xml:lang="he">אפליקציית YouTube בקוד פתוח לשמירה על פרטיות</summary>
+  <summary xml:lang="hi">गोपनीयता के लिए एक ओपन-सोर्स यूट्यूब ऐप</summary>
+  <summary xml:lang="hr">YouTube aplikacija otvorenog koda za privatnost</summary>
+  <summary xml:lang="hu">Nyílt forráskódú YouTube-alkalmazás az adatvédelem érdekében</summary>
+  <summary xml:lang="hy">Բաց կոդով YouTube հավելված՝ գաղտնիության համար</summary>
+  <summary xml:lang="id">Aplikasi YouTube sumber terbuka untuk privasi</summary>
+  <summary xml:lang="is">Opinn YouTube app fyrir friðhelgi einkalífsins</summary>
+  <summary xml:lang="it">Un'app YouTube open source per la privacy</summary>
+  <summary xml:lang="ja">プライバシー保護のためのオープンソースのYouTubeアプリ</summary>
+  <summary xml:lang="kn">ಗೌಪ್ಯತೆಗಾಗಿ ಓಪನ್ ಸೋರ್ಸ್ YouTube ಅಪ್ಲಿಕೇಶನ್</summary>
+  <summary xml:lang="ko">개인 정보 보호를 위한 오픈 소스 YouTube 앱</summary>
+  <summary xml:lang="lt">Atvirojo kodo „YouTube“ programėlė privatumui užtikrinti</summary>
+  <summary xml:lang="lv">Atvērtā koda YouTube lietotne privātuma nodrošināšanai</summary>
+  <summary xml:lang="nl">Een open-source YouTube-app voor privacy</summary>
+  <summary xml:lang="no_NY">En YouTube-app med åpen kildekode for personvern</summary>
```

Comment:
I dont see this anywhere on https://flathub.org/languages. Remove all translations related to this lang